docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -311,6 +311,13 @@ Along with the HTTP error code, our API can also return more detailed error code
 | 240000 | Message blocked by harmful links filter                                                                                       |
 | 350000 | Cannot enable onboarding, requirements are not met                                                                            |
 | 350001 | Cannot update onboarding while below requirements                                                                             |
+| 520000 | Poll voting blocked                                                                                                           |
+| 520001 | Poll expired                                                                                                                  |
+| 520002 | Invalid channel type for poll creation                                                                                        |
```

Comment:
Maybe this limitation(?) could be mentioned somewhere (e.g. in the poll section)...

Reply:
I'll keep this comment open, but we're also revisiting the announcement channel restriction soon-ish (I assume you basically mean that)

Reply:
I'm pretty confident polls are allowed in announcement channels now.

Reply:
Are there still cases where this error code is returned?

Reply:
> Are there still cases where this error code is returned?

@tpcstld